src/server.c

```diff
@@ -124,6 +158,87 @@ request_type_t request_paths[] = {
     {REQ_POST, "/v1/log", SERTY_BOTH, &handleCloudLog},
     {REQ_POST, "/v1/cloud-reset", SERTY_BOTH, &handleCloudReset}};
 
+error_t handleCacheDownload(HttpConnection *connection, const char_t *uri, const char_t *queryString, client_ctx_t *client_ctx)
```

Comment:
this should be (later) reworked to use real api calls and not "url" controlled actions?

like api for 
* clearCache 
* getCacheStats (returns a json with the values, no html)

Reply:
Edit: solved